src/arrayMethodSort.js

```diff
@@ -4,9 +4,43 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultFunction) {
+    let check = 0;
+
+    do {
+      check = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prevEl = this[i - 1];
+        const currEl = this[i];
+        const comparison = compareFunction(prevEl, currEl);
+
+        if (comparison > 0) {
+          this[i - 1] = currEl;
+          this[i] = prevEl;
+          check++;
+        }
+      };
+    } while (check > 0);
+
+    return this;
   };
 }
 
+function defaultFunction(prevEl, currEl) {
+  const prevStr = prevEl.toString();
+  const currStr = currEl.toString();
+
+  switch (true) {
+    case prevStr > currStr:
+      return 1;
+
+    case prevStr === currStr:
+      return 0;
+
+    case prevStr < currStr:
+      return -1;
+  }
```

Comment:
```suggestion
  if (prevStr > currStr:) {
    return 1;
  }
  
  if (prevStr < currStr:) {
    return -1;
  }
  
  return 0;
```

we can use if statement in this case